lib/tasks/analyse/links.rb

```diff
@@ -52,16 +52,15 @@ def write_overview_worksheet(results, spreadsheet)
     overview_worksheet = spreadsheet.worksheets[0]
     overview_worksheet.title = 'Overview'
 
-    add_results_to_worksheet(results, overview_worksheet)
+    add_results_to_worksheet(results, overview_worksheet, "accordion")
```

Comment:
Thanks for your feedback @alecgibson, passing in the column names is much better. I have updated the changes to this commit.